### DISCUSSION TITLE
Plotting Positive/Negative Pole Figures and 2 or more data sets.

### DISCUSSION THREAD

**Discussion by beroshcr** [mtex-toolbox/mtex#1712] [Ask Anything]
Hi,

How does one alter the type of stereographic projection used for pole figures?

Also, I have multiple data sets of the same type with crystals of identical symmetry. Is it possible to plot more than data set on a singular pole figure?

Thank you.

**Comment by kilir** [comment]
Hi,
while there's only one "type of stereographic projection" you can switch from an equal area projection (Lambert or Schmid net) to an equal angle (stereographic or Wulff net) using `plot(...., 'projection','eangle')` . There's also orthographic, plain .....

To plot multiple things in on spherical projection, use `hold on` `hold off` and evetualle add `'add2all'` as an option for subsequent plots.

Hope this helps.
Rüdiger
